src/main/resources/defaults/conversational-search-defaults.json

```diff
@@ -0,0 +1,20 @@
+{
+    "template.name": "deploy-cohere-chat-model",
+    "template.description": "deploying cohere chat model",
```

Comment:
nit - `A template to deploy a Cohere chat model`. I think we should try to be clean with metadata / descriptions, as we will probably use this to bubble up on the frontend for future OOTB use cases.

Reply:
makes sense, do you think if we mention defaults might change in other releases its okay to change these later without worrying on bwc between templates? At least for simple things like if we want to change descriptions later

Reply:
Sure, but I think we should at least clean up the metadata/description stuff now. Quick minor changes but looks much more presentable. Adding details etc. we can change later, as we'll for sure already need to make schema changes.

Reply:
Yup, didn't mean instead of now, just made me though of if we need to do more in future, did some more clean up now